S3/English/ASS_25.ass

```diff
@@ -0,0 +1,117 @@
+Dialogue: 0,0:01:09.10,0:01:10.64,Default,,0,0,0,,How's it going in hell
+Dialogue: 0,0:01:11.06,0:01:12.43,Default,,0,0,0,,I'm fine living here
+Dialogue: 0,0:01:13.61,0:01:14.38,Default,,0,0,0,,Recently
+Dialogue: 0,0:01:14.78,0:01:16.26,Default,,0,0,0,,There's someone who I care about
+Dialogue: 0,0:01:16.37,0:01:17.21,Default,,0,0,0,,I'm coming
+Dialogue: 0,0:01:17.33,0:01:19.54,Default,,0,0,0,,She's the person I like
+Dialogue: 0,0:01:20.75,0:01:22.89,Default,,0,0,0,,Though I haven't expressed her my feelings to her yet
+Dialogue: 0,0:01:23.45,0:01:25.83,Default,,0,0,0,,But I'm satisfied with how it is
+Dialogue: 0,0:01:26.00,0:01:27.43,Default,,0,0,0,,Since I get to spend all my time with her anyway
+Dialogue: 0,0:01:28.25,0:01:28.97,Default,,0,0,0,,Besides
+Dialogue: 0,0:01:29.40,0:01:32.71,Default,,0,0,0,,The dark lord inside me has released most of his energy
+Dialogue: 0,0:01:33.13,0:01:35.52,Default,,0,0,0,,The Dark Force Union could no longer sense his existence
+Dialogue: 0,0:01:36.47,0:01:38.92,Default,,0,0,0,,I'm not a timebomb anymore
+Dialogue: 0,0:01:38.96,0:01:41.79,Default,,0,0,0,,Who still write diaries anymore
+Dialogue: 0,0:01:42.80,0:01:43.79,Default,,0,0,0,,Not a diary!
+Dialogue: 0,0:01:43.96,0:01:45.24,Default,,0,0,0,,It's an email!
+Dialogue: 0,0:01:45.31,0:01:45.70,Default,,0,0,0,,No
+Dialogue: 0,0:01:45.70,0:01:46.73,Default,,0,0,0,,Don't look at it!
+Dialogue: 0,0:01:47.02,0:01:48.65,Default,,0,0,0,,Didn't we agreed you should stay inside me
+Dialogue: 0,0:01:48.82,0:01:50.24,Default,,0,0,0,,and don't bother each other
+Dialogue: 0,0:01:53.74,0:01:55.92,Default,,0,0,0,,Girls don't like nagging boys
+Dialogue: 0,0:01:56.15,0:01:57.50,Default,,0,0,0,,You... be careful
+Dialogue: 0,0:01:59.39,0:01:59.99,Default,,0,0,0,,Lily
+Dialogue: 0,0:02:00.22,0:02:01.04,Default,,0,0,0,,Lily is back!
+Dialogue: 0,0:02:01.23,0:02:01.73,Default,,0,0,0,,Lily
+Dialogue: 0,0:02:03.00,0:02:03.61,Default,,0,0,0,, {\pos(970,200)} Lily
+Dialogue: 0,0:02:02.86,0:02:04.15,Default,,0,0,0,,Beware of others who are ahead of you
+Dialogue: 0,0:02:08.86,0:02:10.33,Default,,0,0,0,,They still didn't made peace?
+Dialogue: 0,0:02:10.97,0:02:12.11,Default,,0,0,0,, {\pos(970,200)}You're back...
+Dialogue: 0,0:02:13.92,0:02:14.25,Default,,0,0,0,,Oh
+Dialogue: 0,0:02:14.49,0:02:15.88,Default,,0,0,0,,How are classmates in heaven doing?
+Dialogue: 0,0:02:16.66,0:02:17.88,Default,,0,0,0,,Everyone's the same as before
+Dialogue: 0,0:02:18.82,0:02:19.06,Default,,0,0,0,,Lily
+Dialogue: 0,0:02:19.68,0:02:21.11,Default,,0,0,0,,You have no idea how much I've missed you
+Dialogue: 0,0:02:21.56,0:02:23.16,Default,,0,0,0,,The class got so boring since you left
+Dialogue: 0,0:02:24.00,0:02:25.82,Default,,0,0,0,,Maybe you should stay for a few more days
+Dialogue: 0,0:02:26.17,0:02:27.69,Default,,0,0,0,,I made lots of interesting friends
+Dialogue: 0,0:02:28.00,0:02:29.20,Default,,0,0,0,,You'll like them
+Dialogue: 0,0:02:32.13,0:02:33.20,Default,,0,0,0,,It's Neil
```

Comment:
"That's" instead of it's if she's introducing him, "there's" if she's connecting it to the previous sentence where she mentions her interesting friends

Reply:
The original line was more like "Oh isn't this Neil" I think

Reply:
they were cut in their discussion because of his whining, so it's more like a surprise. I think "It's Neil" or "Oh, it's Neil" fits

Reply:
ah yeah, "Oh, it's Neil!" then I think